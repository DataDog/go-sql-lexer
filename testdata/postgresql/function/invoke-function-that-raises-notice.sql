SELECT log_activity('User logged in');
