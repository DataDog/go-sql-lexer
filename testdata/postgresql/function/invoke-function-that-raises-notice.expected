[
    {
      "expected": "SELECT log_activity ( ? )"
    }
  ]
  